packages/utils/src/abbreviate-number/abbreviate-number.ts

```diff
@@ -0,0 +1,8 @@
+export const abbreviateNumber = (n: number) => {
+  if (n < 1e3) return n.toString();
+  if (n >= 1e3 && n < 1e6) return +(n / 1e3).toFixed(2) + 'K';
+  if (n >= 1e6 && n < 1e9) return +(n / 1e6).toFixed(2) + 'M';
+  if (n >= 1e9 && n < 1e12) return +(n / 1e9).toFixed(2) + 'B';
+  if (n >= 1e12) return +(n / 1e12).toFixed(2) + 'T';
+  return n.toString();
+};
```

Comment:
```suggestion
export function abbreviateNumber(n: number) {
  if (n < 1e3) return n.toString();
  if (n >= 1e3 && n < 1e6) return +(n / 1e3).toFixed(2) + 'K';
  if (n >= 1e6 && n < 1e9) return +(n / 1e6).toFixed(2) + 'M';
  if (n >= 1e9 && n < 1e12) return +(n / 1e9).toFixed(2) + 'B';
  if (n >= 1e12) return +(n / 1e12).toFixed(2) + 'T';
  return n.toString();
}
```

🙃

Reply:
I'll change, thanks! It's just lifted from the extension. Good old Thomas 

![Screenshot 2024-09-17 at 11 16 25](https://github.com/user-attachments/assets/0253610b-c16f-4f81-b3fb-fc16ffd6a6fc)